cs/src/Management/TunnelManagementClient.cs

```diff
@@ -1249,11 +1287,16 @@ public async Task<NamedRateStatus[]> ListUserLimitsAsync(CancellationToken cance
             return await this.SendRequestAsync<bool>(
                 HttpMethod.Get,
                 clusterId: null,
-                TunnelsApiPath + "/" + name + CheckAvailableSubPath,
+                (string.IsNullOrEmpty(ApiVersion) ? TunnelsV1ApiPath : TunnelsApiPath) + "/" + name + CheckAvailableSubPath,
                 query: null,
                 options: null,
                 cancellation
             );
         }
+
+        private string? GetApiVersionQuery()
```

Comment:
Perhaps name it just `GetApiQuery` and make virtual and protected? In case more query parameters are needed.

Reply:
updated